### ISSUE TITLE
No right click menu on tray items

### ISSUE THREAD

**Issue by ThatOneCalculator** [Alexays/Waybar#1619]
I might be doing something wrong, but I'm not getting any any right-click/context menus on tray items.

From my config:
```
  "tray": {
    "icon-size": 14,
    "spacing": 10
  },
```

**Comment by Narice** [comment]
I this this is the same issue as #1494 of #1850
Is it still happening for you?

**Comment by Alexays** [comment]
Right-click context menus for tray items work on master: a right click (button 3) opens the item's dbusmenu, or invokes the SNI `ContextMenu` method when the app provides one; a multi-monitor coordinate bug affecting the popup position was also fixed (commit 65dfabc4). Closing as resolved — if a specific tray app still shows no menu on a recent build from master, please reopen and mention the app.
